### ISSUE TITLE
Crash report

### ISSUE THREAD

**Issue by jessej-samuel** [crashinvaders/gdx-texture-packer-gui#112]
<details><summary>Application Log</summary>

```
Version: 4.10.0
OS: Windows 10 10.0 amd64
JRE: 15.0.2 Oracle Corporation
GPU: Intel(R) HD Graphics 5500
OpenGL vendor: Intel
OpenGL version: 4.4.0 - Build 20.19.15.5063
[ExtensionModuleRepositoryService] Cached data is outdated.
[ExtensionModuleRepositoryService] Requesting new data from remote server.
[ConfigurationController] Incremental fonts are disabled
[VisUI] Warning, using invalid libGDX version for VisUI 1.4.8.
You are using libGDX 1.9.11 but you need 1.9.12. This may cause unexpected problems and runtime exceptions.
[GlobalShortcutHandler] Parsing shortcut file: hotkeys_default.txt
[ExtensionModuleRepositoryService] Data was loaded from remote server.
[text-red]Exception occurred:[] The atlas output dir (D:/Codes/libGDX/SpaceShooter/android/assets) is matching to one of the input files/dirs (D:/Codes/libGDX/SpaceShooter/android/assets/enemyRed4.png). That might lead to an accidental source image loss. Please use separate output directory to write your atlases.
[text-red]Stack trace:[] 
java.lang.IllegalStateException: The atlas output dir (D:/Codes/libGDX/SpaceShooter/android/assets) is matching to one of the input files/dirs (D:/Codes/libGDX/SpaceShooter/android/assets/enemyRed4.png). That might lead to an accidental source image loss. Please use separate output directory to write your atlases.
	at com.crashinvaders.texturepackergui.controllers.packing.processors.DataValidationProcessor.processPackage(DataValidationProcessor.java:47)
	at com.crashinvaders.texturepackergui.utils.packprocessing.CompositePackProcessor.processPackage(CompositePackProcessor.java:20)
	at com.crashinvaders.texturepackergui.utils.packprocessing.PackProcessingManager.lambda$execute$0(PackProcessingManager.java:60)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:832)

[text-red]Exception occurred:[] The atlas output dir (D:/Codes/libGDX/SpaceShooter/android/assets) is matching to one of the input files/dirs (D:/Codes/libGDX/SpaceShooter/android/assets/enemyRed4.png). That might lead to an accidental source image loss. Please use separate output directory to write your atlases.
[text-red]Stack trace:[] 
java.lang.IllegalStateException: The atlas output dir (D:/Codes/libGDX/SpaceShooter/android/assets) is matching to one of the input files/dirs (D:/Codes/libGDX/SpaceShooter/android/assets/enemyRed4.png). That might lead to an accidental source image loss. Please use separate output directory to write your atlases.
	at com.crashinvaders.texturepackergui.controllers.packing.processors.DataValidationProcessor.processPackage(DataValidationProcessor.java:47)
	at com.crashinvaders.texturepackergui.utils.packprocessing.CompositePackProcessor.processPackage(CompositePackProcessor.java:20)
	at com.crashinvaders.texturepackergui.utils.packprocessing.PackProcessingManager.lambda$execute$0(PackProcessingManager.java:60)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:832)

java.lang.IllegalArgumentException: actor cannot be null.
	at com.badlogic.gdx.scenes.scene2d.ui.Table.getCell(Table.java:417)
	at com.kotcrab.vis.ui.widget.MenuItem.getImageCell(MenuItem.java:352)
	at com.kotcrab.vis.ui.widget.file.internal.AbstractSuggestionPopup.createMenuItem(AbstractSuggestionPopup.java:36)
	at com.kotcrab.vis.ui.widget.file.internal.DirsSuggestionPopup$1$1.run(DirsSuggestionPopup.java:88)
	at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.loop(Lwjgl3Application.java:157)
	at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.<init>(Lwjgl3Application.java:117)
	at com.crashinvaders.texturepackergui.desktop.ApplicationStarter.start(ApplicationStarter.java:56)
	at com.crashinvaders.texturepackergui.desktop.ApplicationStarter.main(ApplicationStarter.java:35)
```
</details>

I clicked backspace while typing the path for the output dir.

**Comment by metaphore** [comment]
Thanks for the report. 
The bug was fixed in eab8765a089e0736f0d5c1ab9399faf2898bc85b. 
Please wait for the next release.

**Comment by jessej-samuel** [comment]
Sure, welcome.
